WordPress/Classes/ViewRelated/Reader/WPImageViewController.h

```diff
@@ -4,13 +4,17 @@
 
 @interface WPImageViewController : UIViewController
 
+@property (nonatomic, weak) NSNumber *index;
+
 - (instancetype)initWithImage:(UIImage *)image;
 - (instancetype)initWithURL:(NSURL *)url;
 - (instancetype)initWithMedia:(Media *)media;
+- (instancetype)initForGallery:(NSURL *)url andIndex:(NSNumber *)index;
```

Comment:
If we ditch the `index` property as mentioned above, we can then just use the `initWithURL:` initialiser and lose this one. If we _did_ keep this one, I'd rename it to `initWithGalleryURL:imageIndex:` for consistency, but that's a little confusing as the URL is just for an image, not an entire gallery.